### ISSUE TITLE
Performance on Russian

### ISSUE THREAD

**Issue by TungyuYoung** [yangb05/PengChengStarling#2]
感谢您很棒的工作！请问俄语上的性能表现如何？WER为多少？

**Comment by yangb05** [comment]
Thank you for your attention. The WER for Russian on our internal test set is 16.73, while Whisper-Large v3 achieves a result of 12.66 on the same test set, for your reference.

**Comment by TungyuYoung** [comment]
Thanks for your reply. Is there any possible way to reduce the WER? By the way, how's it on English?

**Comment by yangb05** [comment]
To improve Russian performance, fine-tuning the model with a high-quality Russian ASR corpus and a relatively small learning rate could be beneficial. However, since the fine-tuning process is highly sensitive to both the corpus and learning rate, we have yet to find a stable approach.

For English, our results are not ideal, likely due to the limited training data—we used only 2,000 hours from the GigaSpeech corpus, resulting in a WER of 21.06 on the GigaSpeech test set.
